classes/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id, :parent_permission
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Uknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] [Optional] Any `public` method can be added after the `initialize` method without adding the word `public` before it.
So please add the `can_use_services?` method right after the `initialize` method without the word public.